### ISSUE TITLE
[SF][A11y][switch] role should be 'switch'

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#4633]
### **Bug Description**
The role of switch is 'checkbox' (https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Switch.hbs#L3)

### **Expected Behavior**
The role of switch should be switch.
References:
https://w3c.github.io/aria-practices/examples/switch/switch.html
https://ui5.sap.com/#/entity/sap.m.Switch/sample/sap.m.sample.Switch

### **Context**
- UI5 Web Components version: 1.0.2
- OS/Platform: macOS
- Browser: Chrome
- Affected component: switch

**Comment by LidiyaGeorgieva** [comment]
Hello, @SAP/ui5-webcomponents-topic-b, please have a look at this issue.

**Comment by ilhan007** [comment]
Hello @2bno1 Mintao, the role has been changed and merged to the master. We will update you once it's available. 
Regular release (1.2.0) would be at the end of February, but it's possible to release interim patches earlier..
